File: 5.py
magic_number = 20
num = magic_number

while(True):
	success = True	
	for i in reversed(range(1, magic_number)):
		if num % i != 0:
			success = False
			break
		
	if success:
		print "Answer is: %i" % num
		break
	num += magic_number

File: 5.c
#include <stdio.h>

int main()
{
	int magicNumber = 20;
	int num = magicNumber;

	while(true)
	{
		bool success = true;
		for(int i = magicNumber-1; i > 0; i--)
		{
			if(num % i != 0)
			{
				success = false;
				break;
			}
		}
			
		if(success)
		{
			printf("Answer is: %i", num);
			break;		
		}
		num += magicNumber;
	}
	return 0;

}
